### PR TITLE
Patch auto click golden cookie for v2.002

### PR DESCRIPTION
**Pull request by petriewong**

### CODE REVIEW

**Comment by builtinnya** [comment]
Hi @petriewong 
Thank you for your PR to fix #4 !
I will look into the PR as soon as possible. Please be patient 🙇

File: src/js/modules/gameClient.js [hunk 1]
```diff
@@ -64,9 +64,17 @@ var gameClient = function (WatchJS, Game, pageMessagingClient) {
   };
 
   var clickGoldenCookie = function () {
-    if (Game.goldenCookie.wrath > 0 && redCookieAvoidance)
-      return;
-    Game.goldenCookie.click();
+    if (Game.shimmers.length <= 0) return;
+    Game.shimmers.forEach(function (goldenCookie) {
```
Comment (Gray-Wind) [inline src/js/modules/gameClient.js]: For each golden cookie?

File: src/js/modules/gameClient.js [hunk 2]
```diff
@@ -87,8 +95,8 @@ var gameClient = function (WatchJS, Game, pageMessagingClient) {
   };
 
   var goldenCookieWatcher = function (prop, action, newValue, oldValue) {
-    if (oldValue === 0 && newValue > 0)
-      clickGoldenCookie();
+    if (newValue > oldValue && newValue > 0 && Game.shimmers[Game.shimmers.length - 1].type == "golden")
```
Comment (Gray-Wind) [inline src/js/modules/gameClient.js]: If golden cookie in the array (not only last element).

**Comment by Gray-Wind** [comment]
👍 LGTM

File: src/js/modules/gameClient.js (hunk 2 above)
Comment (petriewong) [inline src/js/modules/gameClient.js]: Thank you! Fixed those two issues in d26ffa8.

**Comment by builtinnya** [comment]
@Gray-Wind Thanks! You helped me a lot twice!
@petriewong LGTM 👍  May I merge the PR?

**Comment by petriewong** [comment]
Sure!

**Comment by builtinnya** [comment]
I've just published v0.5.2.
Thank you guys!

**Pull request merged**